### PR TITLE
add: `touchegg-git`

### PR DESCRIPTION
**Pull request by Zahrun**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [ ] Add maintainer to pacscript

### CODE REVIEW

File: packages/touchegg-git/touchegg-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+build_depends="git build-essential gdb cmake debhelper libudev-dev libinput-dev libpugixml-dev libcairo2-dev libx11-dev libxtst-dev libxrandr-dev libxi-dev libglib2.0-dev libgtk-3-dev"
+breaks="${pkgname} ${pkgname}-bin ${pkgname}-deb ${pkgname}-app"
+description="Touchégg is an app that runs in the background and transform the gestures you make on your touchpad or touchscreen into visible actions in your desktop."
+maintainer="Zahrun <Zahrun@github.com>"
+pkgver() {
+  git ls-remote "${url}" refs/heads/master | cut -f1 | cut -c1-8
+}
+version="$(pkgver)"
+
+prepare() {
```
Comment (Henryws) [inline packages/touchegg-git/touchegg-git.pacscript:13]: You can remove this function, as it’s not used

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
CI works, it just can’t connect to dbus.

File: packages/touchegg-git/touchegg-git.pacscript (hunk 1 above)
Comment (Zahrun) [reply packages/touchegg-git/touchegg-git.pacscript]: Great, thanks

File: packages/touchegg-git/touchegg-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+version="$(pkgver)"
+
+build() {
+  mkdir build && cd build
+  cmake -DCMAKE_BUILD_TYPE=Release ..
+  make -j"$(nproc)"
+}
+
+install() {
+  sudo sh -c "DESTDIR=${pkgdir} make install"
```
Comment (Henryws) [inline packages/touchegg-git/touchegg-git.pacscript:20]: Why can't you just run make normally?

Comment (Zahrun) [reply packages/touchegg-git/touchegg-git.pacscript]: Sorry I didn’t know where to put the DESTDIR, now corrected

**Review by Henryws — APPROVED** [review]

**Pull request merged**